### PR TITLE
ENT-12142: Updated docs diagrams 

### PR DESCRIPTION
**Pull request by chiachenglu**
Updated the diagrams 

Ticket: [ENT-12141](https://northerntech.atlassian.net/browse/ENT-12142)

[ENT-12141]: https://northerntech.atlassian.net/browse/ENT-12141?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by olehermanse** [comment]
@cf-bottom please build this in Jenkins :)

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-master/608/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-master/608/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-master-608/output/_site/

**Review by olehermanse — APPROVED** [review]

File: resources/faq/how-does-cfengine-work-agent-workflow.png
Comment (nickanderson) [inline resources/faq/how-does-cfengine-work-agent-workflow.png:1]: Here it shows commuting policy files and defining state directly on the hub, generally I would show that as a separate repository that the hub itself interfaces with but that might not be important in this specific image.

**Review by nickanderson — COMMENTED** [review]
These are good improvements.

File: resources/faq/how-does-cfengine-work-agent-workflow.png
Comment (nickanderson) [reply resources/faq/how-does-cfengine-work-agent-workflow.png:1]: Show the cosmic rays

**Comment by nickanderson** [comment]
I already backported this 

- https://github.com/cfengine/documentation/pull/3338
- https://github.com/cfengine/documentation/pull/3337

**Pull request merged**